src/main/scala/tilelink/Broadcast.scala

```diff
@@ -28,6 +34,94 @@ case class TLBroadcastParams(
   require (numTrackers > 0)
 }
 
+/** [[TLBroadcast]] is the last level coherence manger for TileLink. Simplest topology can be this:
+  * {{{
+  *   A           B
+  *   │           │
+  *   └─────┬─────┘
```

Comment:
This is also an Xbar, otherwise A cannot probe B.

Reply:
I think you need to be really very clear in this diagram about whether A and B are different banks or different masters.

Reply:
Sorry missed xbar in this diagram, I'll fix it tomorrow.